cerbero/utils/git.py

```diff
@@ -290,7 +290,7 @@ async def local_checkout(git_dir, local_git_dir, commit, logfile=None):
     # missing
     branch_name = 'cerbero_build'
     shell.call('%s reset --hard %s' % (GIT, commit), local_git_dir, logfile=logfile)
-    shell.call('%s branch %s' % (GIT, branch_name), local_git_dir, fail=False, logfile=logfile)
+    shell.call('%s branch %s %s' % (GIT, branch_name, commit), local_git_dir, fail=False, logfile=logfile)
```

Comment:
Why did you need to add the commit here? I'm guessing because you need to ensure the commit here, but I wonder if this may cause any regression because the commit itself was never used. Maybe `cerbero_build` was always used for a reason. In any case, commit doesn't seem to be a boolean, so we need to update the docu for the function